### ISSUE TITLE
[ Profile Customization ] [ Drow ] Custom Set not Working

### ISSUE THREAD

**Issue by crimsonvspurple** [ValveSoftware/Dota2-Gameplay#14662]
### Description

Selecting Drow custom set with arcana shows up fine while editing profile but upon hitting save, it switches to some other set.

Selecting Drow arcana set directly works fine.

This applies for mini profile too.

### Screenshots

Preview working with custom set with arcana:
<img width="447" alt="image" src="https://github.com/ValveSoftware/Dota2-Gameplay/assets/11375716/07efc8c3-d83e-4692-bfba-065cb4b9c54f">

Changed to some other set after hitting save!

<img width="338" alt="image" src="https://github.com/ValveSoftware/Dota2-Gameplay/assets/11375716/35027b1e-e956-4fac-9fc1-f6d7c4f64795">

Working with arcana set:
<img width="378" alt="image" src="https://github.com/ValveSoftware/Dota2-Gameplay/assets/11375716/2d1243c5-87ba-4090-ad6a-f978c3ae003f">

Basically:
First one doesn't work but 2nd one works.

<img width="1388" alt="image" src="https://github.com/ValveSoftware/Dota2-Gameplay/assets/11375716/66f7de01-e66b-4e54-9bc0-be6964e0b33a">
